src/handlers/dashboard-extract/extract-data-reformatter.ts

```diff
@@ -0,0 +1,39 @@
+import { ResultSet } from "@aws-sdk/client-athena";
+
+export class ExtractDataReformatter {
```

Comment:
I actually modelled this after your QueryExecutionValidator, as it made mocking the Athena calls a lot easier I think.  But I think maybe it can be removed now after switching to the context handler approach.  Will do that now.